lib/util.js

```diff
@@ -2,7 +2,7 @@
 
 const BBPromise = require('bluebird');
 const express = require('express');
-const uuid = require('cassandra-uuid');
+const uuidV1 = require('uuid/v1');
```

Comment:
Let's use all lowercases here, i.e. `uuidv1`

Reply:
Changed to lowercase